ZuluContent/Engines/Harvest/Mining.cs

```diff
@@ -89,8 +87,101 @@ private Mining()
 
                 BonusEffect = HarvestBonusEffect
             };
-
+                
             Definitions.Add(oreAndStone);
+            
+            var sand = new HarvestDefinition
```

Comment:
Whilst Sand/Clay mining has identical properties in some cases to Ores its best to have them configured independently for economical purposes. e.g. You may want to alter the amount of Sand being mined to control how many empty bottles are made etc.

Reply:
Yeah I was thinking about that too then wondering if it was worth it, but can add seperate configs for them